src/app/loadout-builder/filter/LockArmorAndPerks.tsx

```diff
@@ -260,14 +260,6 @@ export default memo(function LockArmorAndPerks({
           </button>
         </div>
       </div>
-      {/* Armor Upgrades */}
-      {isPhonePortrait && (
-        <div className={styles.guide}>
-          <ol start={3}>
```

Comment:
There's a corresponding `<ol start={4}>` that would need to be downgraded to `3` in the main `LoadoutBuilder` component for the phone layout.